src/Scene.ts

```diff
@@ -682,18 +682,27 @@ export class Scene {
     this.COMvisual.add(mesh);
   }
 
-  public addSky(): void {
+  public addSky(cubemap: string | undefined): void {
     var cubeLoader = new THREE.CubeTextureLoader();
```

Comment:
If this asset comes from the websocket, we should pass the websocket loading manager here as well. Check for other cases where it was used.

Reply:
> I'm assuming that I'll add all skyboxes to fuel. I'd that okay?

It looks like this was already done, correct?  That is, if I use this branch and add in a `cubemap_uri` to my SDF, I get a red sky as desired.  So I think that means everything was properly downloaded from fuel.  Do I have that right?

Reply:
Yes, that sounds right